tools/topology/topology2/platform/intel/dmic-wov.conf

```diff
@@ -0,0 +1,172 @@
+Object.Pipeline {
+	passthrough-capture.1 {
+		format		$FORMAT
+		rate		16000
+		rate_min	16000
+		rate_max	16000
+		index 		$DMIC1_HOST_PIPELINE_ID
+		Object.Widget.pipeline.1 {
+			stream_name $DMIC1_PIPELINE_STREAM_NAME
+		}
+		Object.Widget.copier.1 {
+			stream_name $DMIC1_PCM_CAPS
+			Object.Base.audio_format.1 {
+				in_rate			16000
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_rate		16000
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$ibs * 2]"
+			}
+			Object.Base.audio_format.2 {
+				in_rate			16000
+				in_channels		4
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_rate		16000
+				out_channels		4
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$ibs * 2]"
+				in_ch_cfg	$CHANNEL_CONFIG_3_POINT_1
+				in_ch_map	$CHANNEL_MAP_3_POINT_1
+				out_ch_cfg	$CHANNEL_CONFIG_3_POINT_1
+				out_ch_map	$CHANNEL_MAP_3_POINT_1
+			}
+		}
+	}
+
+	passthrough-be.12 {
```

Comment:
@libinyang even though this works. This is incorrect. What you need is a new pipeline for kpb-capture that has the DAI copier and the KPB connected

Reply:
> @libinyang even though this works. This is incorrect. What you need is a new pipeline for kpb-capture that has the DAI copier and the KPB connected

Why do we need a different pipeline for kbp and DAI? The dmic-wov is only used for wov and I prefer it not to be re-used by other scenarios.